go-update/src/main/java/org/reactome/release/goupdate/GoUpdateStep.java

```diff
@@ -50,19 +51,24 @@ public void executeStep(Properties props) throws SQLException
 			// 5) delete the marked-for-deletion instances.
 			// 6) update relationships between remaining instances, based on content of data structure.
 			
+			// ***UPDATE***
+			// URL to main GO file is now:
+			// http://current.geneontology.org/ontology/go.obo
+			// ...the ec2go file is the same.
+			
 			MySQLAdaptor adaptor = getMySQLAdaptorFromProperties(props);
 			this.loadTestModeFromProperties(props);
 			
-			long personID = new Long(props.getProperty("person.id"));
+			long personID = Long.valueOf(props.getProperty("person.id")).longValue();
```

Comment:
Does the `.longValue()` do something that `Long.valueOf()` doesnt?

Reply:
`Long.valueOf(props.getProperty("person.id")).longValue();` ...in this expression, `Long.valueOf` will return a `Long` whose value is the numeric representation of the string that was passed to it. `longValue()`, when called on a `Long` will return a `long`.

Yes, `long` and `Long` are different. `long` is a primitive, and `Long` is an Object. Primitives are passed by value, objects are passed by reference. Objects can be null, primitives _cannot_ be null. 

More on primitives here: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html
More on numeric versions of primitives here: https://docs.oracle.com/javase/tutorial/java/data/numberclasses.html